llmc/mfu.h

```diff
@@ -39,6 +40,7 @@ typedef struct {
 static GPUEntry gpu_db[] = {
     {"Tesla V100-SXM2-16GB", &VOLTA, 640, 1530},
     {"Tesla V100-PCIE-32GB", &VOLTA, 640, 1530},
+    {"NVIDIA A10", &AMPERE, 72, 885},
```

Comment:
according  to https://www.techpowerup.com/gpu-specs/a10-pcie.c3793, the A10's boost clock is 1695 MHz
using the other ampere reference values, this would lead to ~62 bfloat TFlops, which would be consistent with other ampere cards, are the 125 with sparsity?

Reply:
values were copied from https://www.nvidia.com/content/dam/en-zz/Solutions/Data-Center/a10/pdf/a10-datasheet.pdf . With sparsity, value are doubled to 250 or 500. 885MHz is base clock. I am not aware if it should be boost clock. If I change it to boost clock, it prints "22.5% bf16 MFU".